docs/sources/configuration/_index.md

```diff
@@ -396,7 +396,7 @@ results_cache:
 # Perform query parallelisations based on storage sharding configuration and
 # query ASTs. This feature is supported only by the chunks storage engine.
 # CLI flag: -querier.parallelise-shardable-queries
-[parallelise_shardable_queries: <boolean> | default = false]
+[parallelise_shardable_queries: <boolean> | default = true]
```

Comment:
This is a _big_ thing to change as it increases parallelism by ~16x for current schema defaults. On larger Loki clusters, it'll definitely be an improvement, but it's likely to be _slower_ for smaller ones, which is why we haven't changed this default before. I'm definitely worried for the effects of defaulting this to `true` alongside the `split_queries_by_interval` changes. Perhaps we just change `split_queries_by_interval` and leave `parallelise_shardable_queries=false`? WDYT?

Reply:
hmmm, yeah I did not consider that impact on smaller clusters. I think in general our defaults should be geared for smaller clusters, so I'm in favor with setting this back to false and just keeping the `split_queries_by_interval`

Reply:
@trevorwhitney @owen-d 
Some clusters have an issue with this default setting: https://github.com/grafana/loki/issues/4613

Reply:
@dfoxg commented in #4613, but for visibility, is this still an issue given the default change in #2048?

Reply:
@trevorwhitney I think your link to the pull request is wrong

Reply:
Indeed it was. Fixed